README.md

```diff
@@ -38,7 +45,7 @@ Create #my_inject
 
 To get a local copy up and running follow these simple example steps.
 
-1. Open the terminal comand line window
+1. Open the terminal command line window
 2. `git clone https://github.com/epinczinger/ruby_enumerables.git`
 3. Execute `ruby enumerables.rb`
```

Comment:
For this project, you are not creating an executable file, it doesn't make sense to issue `ruby enumerables.rb`, because this is a module, it doesn't interact with the user in any form.